packages/plugins/experimental/plugin-transcription/src/components/TranscriptContainer.tsx

```diff
@@ -56,16 +60,15 @@ export const TranscriptionContainer: FC<{ transcript: TranscriptType }> = ({ tra
     }
   }, [transcript, edge]);
 
-  // TODO(dmaretskyi): Move action to menu.
   return (
     <StackItem.Content toolbar={false} classNames='relative'>
       <Toolbar.Root classNames='absolute block-start-1 inline-end-1 z-[1] is-min'>
         <IconButton
-          icon='ph--pen--regular'
+          icon='ph--subtitles--regular'
```

Comment:
This is the same icon as the type, I think actions and the entity type should have a distinct icon which is why I chose `pen`. If you don’t like `pen`, I’d still recommend choosing something other than `subtitles` since summaries and transcripts are distinct entity types.

Reply:
oh ok i missed this. i'll change it.